test/cucumber/EditAccount'sCnpj.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Edit account's cnpj
+  As a generator's owner
+  I want to correctly edit my cnpj
+  So that I can keep the system updated
+
+#if($EditGeneratorCnpj)
+  @tagQualquer
+  Scenario: Correctly edit the cnpj
+    Given the system has a residue generator with cnpj "12345678912345"
+    When I change my cnpj to "45912367834512"
+    Then the generator with cnpj "45912367834512" is properly edited
+
+
+  @tagQualquer
+  Scenario: Incorrectly edit the cnpj
+    Given the system has a residue generator with cnpj "12345678912345"
+    When I change my cnpj to "testecnpj"
+    Then the generator with cnpj "testecnpj" is not edited
```

Comment:
evitar o my cnpj

I change my cnpj to "testecnpj" -> I try to change its cnpj to "testecnpj"

        the system doesn't accept the change
And still has a generator with CNPJ "12345678912345"